javascript/javascript1/week1/helloWorld.js

```diff
@@ -1 +1,113 @@
-console.log("HelloWorld");
\ No newline at end of file
+console.log("HelloWorld");
+// Age-ify (A future age calculator)
+
+const yearOfBirth = 1985;
+let yearFuture = 2080;
+let age = yearFuture - yearOfBirth;
+
+
+console.log("You will be " + age + " years old in " + yearFuture)
+
+// Goodboy-Oldboy (A dog age calculator)
+
+let dogYearOfBirth = 1985;
+let dogYearFuture = 2027;
+let dogYear;
+let shouldShowResultInDogYears = true;
+let resultInHumanYears;
+if (dogYear) {
+    shouldShowResultInDogYears = (dogYearFuture - dogYearOfBirth) * 7;
+    console.log(
+        "Your dog will be " + shouldShowResultInDogYears + " dog years old in 2027"
+    );
+
+    //The code is different, but the same as the code above نفس الكود الي فوق بس بصياغة مختلفة
+    
+    /* let dogYearOfBirth = 1985;
+    let dogYearFuture = 2027;
+    
+   
+    const dogAge = dogYearFuture - dogYearOfBirth;
+    const dogYear = dogAge * 7;
+    
+    let shouldShowResultInDogYears = true;
+    
+    if (shouldShowResultInDogYears === true) {
+      console.log(
+        "Your dog will be " + dogYear + " dog years old in " + dogYearFuture + "."
+      );
+    } else {
+      console.log(
+        "Your dog will be " + dogAge + " human years old in " + dogYearFuture + "."
+      );
+    }
+*/
+
+
+   
+//Housey pricey (A house price estimator)
+
+// Peters hous:
+
+const volumeInMeters = 8 * 10 * 10;
+const gardenSizeInM2 = 100;
+const housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const houseCost = 2500000
+
+
+if (housePrice>houseCost) {
+    console.log ("This house is " + (housePrice - houseCost) + "underpriced")
+}
+else {
+    console.log ("This house is " + (houseCost - housePrice) + " overpriced")
+}
+
+// Julias hous:
+
+const volumeInMeters2 = 5 * 11 * 8;
+const gardenSizeInM22 = 70;
+const housePrice2 = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const houseCost2 = 1000000
+
+
+if (housePrice>houseCost) {
+    console.log ("This house is " + (housePrice2 - houseCost2) + " underpriced")
+}
+else {
+    console.log ("This house is " + (housePrice2 - houseCost2) + " overpriced")
+}
```

Comment:
I got "This house is -4879000 overpriced" and it might be nice to return a positive value instead. There are several ways to do it. If you want to learn about the Math object way to do it, look up Math.abs() here https://www.w3schools.com/js/js_math.asp